src/tasks/tasks.service.ts

```diff
@@ -0,0 +1,116 @@
+import { Model } from 'mongoose';
+import { InjectModel } from '@nestjs/mongoose';
+import { HttpStatus, Injectable, Logger } from '@nestjs/common';
+import { CreateTaskDto } from './dto/create-task.dto';
+import { Task, TaskDocument } from './entities/task.entity';
+import {
+  CreateResponse,
+  FindAllResponse,
+  FindOneResponse,
+  RemoveResponse,
+  UpdateResponse,
+} from './dto/proto/task.pb';
+import { FindOneTaskRequest, RemoveTaskRequest } from './dto/task.dto';
+
+@Injectable()
+export class TasksService {
+  constructor(
+    @InjectModel(Task.name) private readonly taskModel: Model<TaskDocument>,
+  ) {}
+
+  private readonly logger = new Logger(TasksService.name);
+
+  public async create(createTaskDto: CreateTaskDto): Promise<CreateResponse> {
+    this.logger.log('Creating new task');
+
+    // TODO - find Skill by id before create Task
+    const task = new this.taskModel(createTaskDto);
+    task.save();
```

Comment:
se não me engano, .save é assincrono.
```suggestion
    await task.save();
```